src/onion.rs

```diff
@@ -46,8 +46,7 @@ impl CustomOnionMessageHandler for OnionMessageHandler {
 		Ok(Some(UserOnionMessageContents { tlv_type: message_type, data: buf.to_vec() }))
 	}
 
-        // We don't need this for our purposes yet.
         fn release_pending_custom_messages(&self) -> Vec<PendingOnionMessage<Self::CustomMessage>> {
-		unimplemented!();
+		vec![]
```

Comment:
Why this change?

Reply:
Was getting an error with unimplemented!(), and saw that vec![] seems to be the de facto implementation for now in rust-lightning's integration tests. I don't *think* we really need a full implementation of release_pending_custom_messages for our tests, at least right now, but I should probably dive in and double check that